Purchases/Networking/Operations/Handling/SubscriberAttributesMarshalling.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SubscriberAttributesMarshalling.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+protocol SubscriberAttributesMarshalling {
```

Comment:
This type of stuff should probably be injected into the operation instead of as a protocol + default implementation.

Reply:
Old, updated